### DISCUSSION TITLE
Bug report: Blocklists aren't working all the time - data added

### DISCUSSION THREAD

**Discussion by bloodyskullz** [AdguardTeam/AdGuardHome#3154] [Q&A]
Hi,

Issue:
Blocklist is not blocking everything stated in the lists. Entries from default blocklists to custom ones added are slipping through the cracks.

Example of domain: 
"doubleclick.net" was processed in IE 11, failed in chrome because of my adblocker "Ublock Origin".

Verified via screenshot below.

Version of adguard home & hardware: v0.106.2 installed on Ubuntu 20.04LTS on an intel NUC 6th gen with 4GB of RAM.

Installed via automated install.

I noticed this issue when my blocklist for adult content wasn't being filtered and blocked correctly. Tested with default list and the issue presented itself again. Please feel free to reach out to me for more information either here or via email mrglitch1989@gmail.com (this is faster). Will be more than happy to assist.

Thanks

![image](https://user-images.githubusercontent.com/79482326/117897340-97fe6b00-b290-11eb-8fd7-5b36c20c65e1.png)

**Comment by ameshkov** [comment]
To troubleshoot this issue we need to see AdGuard Home logs.

1. [Configure](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#verboselog) AdGuard Home to write verbose-level log.
2. Reproduce the issue.
3. Post the log file here.

**Comment by bloodyskullz** [comment]
Where do I find the .yaml file?

**Comment by ameshkov** [comment]
Right near `AdGuardHome` binary file

**Comment by bloodyskullz** [comment]
I am sorry but I don't know which directory this is located. Can you give me a path to the location?

Thanks

**Comment by ameshkov** [comment]
I am not sure how exactly you installed AGH, but if you followed the standard procedure, it would be in `/opt/AdGuardHome`

**Comment by bloodyskullz** [comment]
I used this command to do the install

`curl -sSL https://raw.githubusercontent.com/AdguardTeam/AdGuardHome/master/scripts/install.sh | sh`

Edit: Will the directory still be under /opt/AdGuardHome?

**Comment by EugeneOne1** [comment]
Yes, it should be there, if you're using Linux (Ubuntu in your case).  Also, don't forget to turn _AGH_ off before editing the configuration file, if it isn't.  Otherwise, your changes will be rewritten.

```
cd /opt/AdGuardHome
sudo ./AdGuardHome -s stop
```

You should also turn it on manually afterwards.

```
cd /opt/AdGuardHome
sudo ./AdGuardHome -s start
```

**Comment by bloodyskullz** [comment]
The steps for configuring the yaml file just say to set the log file to "log_file". Where would I be grabbing the info from to paste it here for you when trying this again?

Is the log_file going to be in the same adguard directory mentioned in this thread?

**Comment by bloodyskullz** [comment]
First query is my blocklock failing and letting a request go through. Second one is the doubleclick.net

It seems like my list is completely ignored entirely. Does adguard have a limit to file sizes for block lists?
[github.txt](https://github.com/AdguardTeam/AdGuardHome/files/6469920/github.txt)

**Comment by bloodyskullz** [comment]
Hi,

@ameshkov is info i provided enough? Is it an actual bug?

**Comment by ameshkov** [comment]
@bloodyskullz the log does not seem to be enough.

I see that `www.flameporn.com` is not blocked because it's not in our adult websites list (yet). Is it what you meant? If you encounter a domain that should be listed there, please report it to our support service at `support@adguard.com`, we'll add it promptly.

**Comment by bloodyskullz** [comment]
@ameshkov I'm confused, if I have a blocklist which already contains that site, shouldn't it take effect instantly?

Is there a filesize limit for blocklists?

**Comment by ameshkov** [comment]
There's no limit. Can you share that blocklist?

**Comment by bloodyskullz** [comment]
https://github.com/bloodyskullz/Adguard-blocklist/tree/main/porn%20block

it's a large list so it would be best to download and do a search for the title. I didn't add the www portion as I replicated it based on the default adguard which also didn't contain that.

**Comment by ameshkov** [comment]
You added it to your AGH with `https://raw.githubusercontent.com/bloodyskullz/Adguard-blocklist/main/porn%20block/pornblock.txt` this URL right?

**Comment by bloodyskullz** [comment]
Tes that is right, it still didn't take effect.

**Comment by ameshkov** [comment]
Ah, I see, you're adding spaces which shouldn't be there.

`|| example.org ^` --> `||example.org^`

Also, I see that you're converting simple hosts lists to AG syntax. I suggest you using [this tool](https://github.com/AdguardTeam/HostlistCompiler) for that. It will take care of duplicates, validation, etc, etc.
